packages/vuetify/src/components/VDataTable/__tests__/VDataTable.spec.browser.tsx

```diff
@@ -1,107 +0,0 @@
-// Components
-import { VDataTable } from '../'
-
-// Utilities
-import { render, screen, userEvent } from '@test'
-import { ref } from 'vue'
-
-describe('VDataTable', () => {
-  it('mirrors external updates', async () => {
```

Comment:
These new tests you added don't actually check anything that is likely to ever break and there's lots of tests missing from the originals. I undid these changes so I could merge it, they're back in the branch now though.